packages/patterns/llm.tsx

```diff
@@ -33,6 +33,8 @@ const askQuestion = handler<
 });
 
 export default recipe<LLMTestInput, LLMTestResult>("LLM Test", ({ title }) => {
+  // It is possible to make inline cells like this, but always consider whether it should just be part of the argument cell.
+  // These cells are effectively 'hidden state' from other recipes
```

Comment:
+1

i wonder whether we could have even stronger guidelines here. the only good reason i can imagine is for something like an importer that wants to tightly manage this state, including cells that track e.g. last imported cursors, etc.

so maybe we should advice even stronger against this?